apps/angular/pipe-hard/src/app/app.component.ts

```diff
@@ -1,18 +1,17 @@
 import { NgFor } from '@angular/common';
 import { Component } from '@angular/core';
-import { PersonUtils } from './person.utils';
+import { UtilsPipe } from './utils.pipe';
 
 @Component({
   standalone: true,
-  imports: [NgFor],
+  imports: [NgFor, UtilsPipe],
   selector: 'app-root',
   template: `
-    <div *ngFor="let activity of activities">
+    <div style="padding: 10px" *ngFor="let activity of activities">
       {{ activity.name }} :
       <div
         *ngFor="let person of persons; let index = index; let isFirst = first">
-        {{ showName(person.name, index) }}
-        {{ isAllowed(person.age, isFirst, activity.minimumAge) }}
+        {{ 'isAllowed' | utils : person.age : isFirst : activity.minimumAge }}
```

Comment:
what happened to showName ?

Reply:
Oh gosh, forgot to get it back after little testing 🥲